docker/ol-home-start.sh

```diff
@@ -7,4 +7,5 @@ python --version
 
 echo "Waiting for postgres..."
 until pg_isready --host db; do sleep 5; done
-make reindex-solr
+time make reindex-solr
+echo "You may now open to localhost:8080"
```

Comment:
Some terminals would allow command-click to open the URL.
```suggestion
echo "You may now open http://localhost:8080"
```